acceptance/ui/features/data/default/hosts.json

```diff
@@ -2,49 +2,54 @@
   "hosts": 
   {
     "defaultHost": {
-      "nameAndPort": "roei-dev:4979",
-      "name": "roei-dev",
-      "rpcPort": 4979,
-      "hostID": 4979,
-      "memory": 33681858560,
+      "nameAndPort": "%{local_ip}:4981",
+      "name": "defaultHost",
+      "rpcPort": 4981,
+      "hostID": 4981,
+      "memory": 10737418240,
+      "memoryGB": "10.00 GB",
+      "ramGB": "7.50 GB",
+      "cores": 4981,
+      "kernelVersion": "Default Kernel Version Test",
+      "kernelRelease": "Default Kernel Release Test",
+      "ccRelease": "Default CC Release Test",
       "pool": "default",
-      "commitment": "50%",
-      "outboundIP": "172.17.42.1"
+      "commitment": "75%",
+      "outboundIP": "%{local_ip}"
     },
 
     "host2": {
-      "nameAndPort": "10.87.110.17:4982",
+      "nameAndPort": "%{local_ip}:4982",
       "name": "secondHost",
       "rpcPort": 4982,
       "hostID": 4982,
       "memory": 10737418240,
+      "memoryGB": "10.00 GB",
+      "ramGB": "4.80 GB",
       "cores": 4982,
       "kernelVersion": "Kernel Version test 2",
       "kernelRelease": "Kernel Release test 2",
       "ccRelease": "CC Release test 2",
       "pool": "default",
-      "commitment": "25%",
-      "outboundIP": "10.87.110.17"
+      "commitment": "48%",
```

Comment:
May I ask what the reasoning is behind the selection of these commitment numbers?